app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md

```diff
@@ -0,0 +1,91 @@
+
+Needs also added to the following categories:
+* Recruitment
+  * [Understand the policy around application routes](https://trello.com/c/bp2UPV3m/164-i-need-to-be-aware-of-the-policy-around-application-routes-so-that-i-can-inform-candidates-the-best-route-to-apply-through-and-a)
+* Data and tracking
+  * Repeat need [Download all candidate data as CSV](https://trello.com/c/xB4u69FC/161-i-need-to-download-any-candidate-data-i-need-as-a-csv-so-that-i-can-easily-track-candidates-outside-of-the-manage-system)
+* Course onboarding
+  * [Confirm that a candidate has started on the course](https://trello.com/c/Aizv2IKM/159-i-need-to-confirm-the-candidate-has-started-on-the-course-so-that-my-records-are-correct-and-i-can-report-accurate-data)
+  * [Mark a candidate as having dropped out](https://trello.com/c/OVeYrUWL/160-i-need-to-be-able-to-mark-someone-as-having-dropped-out-of-the-course-before-they-start-and-after-they-start-so-that-my-records)
+  * [Reject a candidate with an offer if they drop out](https://trello.com/c/ajX2HNQ3/165-i-need-to-be-able-to-reject-someone-who-has-accepted-an-offer-if-they-drop-out-so-that-my-records-are-up-to-date)
+  * [Reject/withdraw a deferred offer if the candidate drops out](https://trello.com/c/pgdRucyL/176-i-need-to-reject-withdraw-a-deferred-application-so-that-if-the-candidate-drops-out-my-records-are-up-to-date)
+* Manage data
+  * [Summary statistical data](https://trello.com/c/ErAkkfqg/180-i-need-data-about-all-my-applications-received-rejected-and-offered-throughout-the-year-so-that-i-can-report-accurate-data)
+  * [Data to support DTTP or HESA](https://trello.com/c/7CxOJ0YW/181-i-need-to-receive-the-correct-data-in-order-to-complete-my-hesa-and-dttp-returns-to-the-dfe)
+
+## Findings report
+
+[End of cycle resarch findings](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/edit?usp=sharing)
+* [End of cycle management](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g85c8fec79f_0_157)
+* [Dealing with edge case applications](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_28)
+* [Preparing for the next recruitment cycle](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_35)
+* [Reporting and collecting data](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_42)
+* [COVID-19 Impacts](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g8a6173f9e3_0_128)
+* [Prototype feedback](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g8a6173f9e3_0_198)
+
+## Relates to
+
+* [Sorting by RBD date](../sorting-by-rbd-date)
```

Comment:
We have a mechanism for adding related links, which can be done using the frontmatter, like so:

```
related:
  items:
  - text: Sorting by RBD date
    href: /manage-teacher-training-applications/sorting-by-rbd-date
  - text: Reasons for rejection
    href: /manage-teacher-training-applications/reasons-for-rejection-iteration-3
```

These links will then appear in the sidebar.